### ISSUE TITLE
Autocomplete keyboard navigation shift-tab doesn't work

### ISSUE THREAD

**Issue by m0ksem** [epicmaxco/vuestic-ui#3116]
If you click on input and shift-tab to previous interactive element, the same select will be tabbed.
